stdlib/public/runtime/Private.h

```diff
@@ -183,6 +183,8 @@ namespace swift {
                              void *data);
   uint8_t *_swift_getSectionDataPE(void *handle, const char *sectionName,
                                    unsigned long *sectionSize);
+#endif
+#if defined(__CYGWIN__)
   void _swift_once_f(uintptr_t *predicate, void *context,
```

Comment:
The compiler assumes that `swift_once_t` is zero-initialized, and nonzero after initialization finishes. It doesn't care what exact nonzero values it becomes once initialization begins.